src/server.js

```diff
@@ -42,6 +42,11 @@ app.get('/api/script/:scriptId/characters', scriptController.getCharacters, (req
   return res.status(200).json(res.locals.characters);
 });
 
+// assigns a character to an actor
+app.post('/api/script/:scriptId/characters', scriptController.addCharacter, (req, res) => {
```

Comment:
This url also seems to imply adding a new character since it's posting to script/.../characters. 
Maybe something under actor would make more sense? 
`/api/script/:scriptId/:actorId/characters`,
or `/api/actor/:actorId/:scriptId/characters`,
or `/api/actor/:actorId/assignCharacter`,
or something else?